### PR TITLE
Adding handling of road trip edge case and test

### PR DESCRIPTION
**Pull request by adumortier**
- added the handling of edge case where some `road_trip` query params are missing

### CODE REVIEW

File: app/controllers/api/v1/road_trip_controller.rb [hunk 1]
```diff
@@ -2,12 +2,20 @@ class Api::V1::RoadTripController < ApplicationController
...
+    road_trip = RoadTrip.get_road_trip_info(params[:origin], params[:destination])
+    render json: road_trip.serialize, status: 200
   end
 
-end
\ No newline at end of file
+  private 
+  
+  def missing_query_parameters?
+    query_params[:origin].nil? || query_params[:destination].nil? || query_params[:destination] =='' || query_params[:origin] == ''
```
Comment (adumortier) [inline app/controllers/api/v1/road_trip_controller.rb:14]: checks if query params are missing or empty
